apps/web-desmos/src/screens/proposal_details/components/overview/index.tsx

```diff
@@ -101,8 +109,12 @@ const Overview: FC<{ className?: string; overview: OverviewType }> = ({ classNam
         <Typography variant="body1" className="label">
           {t('type')}
         </Typography>
-        <Typography variant="body1" className="value">
-          {t(type)}
+        <Typography variant="body1">
```

Comment:
Maybe we can remove this parent typography since it is already used  inside the loop